tests/e2e/support/objects/app-files/share/actions.ts

```diff
@@ -95,6 +99,26 @@ export const acceptShare = async (args: ShareStatusArgs): Promise<void> => {
   await page.locator(util.format(filesSharedWithMeAccepted, resource)).waitFor()
 }
 
+export const acceptAllShare = async ({ page }: { page: Page }): Promise<void> => {
+  if (await page.isVisible(showAllButton)) {
+    await page.locator(showAllButton).click()
+  }
+  await page.locator(selecAllCheckbox).click()
+  const numberOfPendingShares = await page.locator(numberOfShares).count()
+  const checkResponses = []
+  for (let i = 0; i < numberOfPendingShares; i++) {
```

Comment:
so, it's done one-by-one. I guess, no batch accept button in webUI

Reply:
we have
![Screenshot 2023-09-27 at 12 07 21](https://github.com/owncloud/web/assets/84779829/35db3e81-6db0-4f10-b162-3630390111b3)

Reply:
Ahh, how I didn't notice it. Then I think, it needs a little bit of strictness because no matter what's the value of `numberOfPendingShares`, if there is one successful response then it assumes all pass.